CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ option(PRECICE_ALWAYS_VALIDATE_LIBS "Validate libraries even after the validatat
 option(PRECICE_BINDINGS_C "Enable the native C bindings" ON)
 option(PRECICE_BINDINGS_FORTRAN "Enable the native Fortran bindings" ON)
 option(PRECICE_BUILD_TOOLS "Build the \"precice-tools\" executable" ON)
+option(PRECICE_BUILD_UNITY "Use unity builds in preCICE" ON)
```

Comment:
At the moment, this only applies to test and I think it would be good to reflect this in the variable description. Also, the name is now similar to BUILD_TOOLS. One might guess that unity is something build by this project, but it is a way to compile objects. So maybe a name like `PRECICE_UNITY_BUILD_TESTING` would be more expressive in my opinion

Reply:
Unit builds are applied to both, preCICE sources and integration tests.

Reply:
Ok then maybe `PRECICE_UNITY_BUILD` would express the difference I am referring to.

Reply:
Why not `PRECICE_CONFIGURE_UNITY_BUILD`?
I think the point about `PRECICE_BUILD_UNITY` been misunderstood as "build something called Unity" is valid.

Reply:
I see what you mean now. It is also called `UNITY_BUILD` in CMake, so that would also be consistent.